quickjs/src/main/java/com/squareup/quickjs/QuickJsException.java

```diff
@@ -64,8 +64,7 @@ private static void addJavaScriptStack(Throwable throwable, String detailMessage
     for (StackTraceElement stackTraceElement : throwable.getStackTrace()) {
       if (!spliced
           && stackTraceElement.isNativeMethod()
-          && stackTraceElement.getClassName().equals(QuickJs.class.getName())
-          && stackTraceElement.getMethodName().equals("evaluate")) {
```

Comment:
What makes this conditional fail?

Reply:
Whoops, this shouldn't have made it in - was just playing. Thanks.